src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyForInitializerPadCheck.java

```diff
@@ -60,6 +74,20 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * for ( ; i &lt; 1; i++ ) { // ok
+ * }
+ *
+ * for (; i &lt; 2; i++ ) {  // violation semicolon is not preceded with whitespace
+ * }
+ *
```

Comment:
Please remove empty lines, to user clearly see space difference

Reply:
done